src/Pages/sanghomoon/Components/MainLeft/Child/Feeds.js

```diff
@@ -0,0 +1,117 @@
+import React from "react";
+import "./Feeds.scss";
+
+class Feeds extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      commentValue: "",
+      commentList: [],
+    };
+  }
+
+  handleComment = (e) => {
+    this.setState({
+      commentValue: e.target.value,
+    });
+  };
+
+  postComment = () => {
+    this.state.commentList.push({ comment: this.state.commentValue });
+    this.setState({
+      commentValue: "",
+    });
+  };
```

Comment:
state는 setState 함수를 활용해서 값을 수정해줘야 다시 render 함수가 호출되고 바뀐 데이터를 반영해서 UI가 다시 render 됩니다. 이런 경우엔 spread 연산자(`...`)를 활용할 수 있습니다. 아래 링크 참고해주세요.

https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Operators/Spread_syntax

```suggestion
  postComment = () => {
    const { commentList, commentValue } = this.state;
    const newCommentList = [...commentList, { comment: commentValue }];
    this.setState({
      commentList: newCommentList
      commentValue: "",
    });
  };
```

Reply:
아 감사합니다! 참고해서 다시 수정해보겠습니다~!